src/Chapter1.hs

```diff
@@ -489,8 +490,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = if n < 0 then mod (n * (-1)) 10 else mod n 10
```

Comment:
Great! You can also use [`abs`](https://hackage.haskell.org/package/base-4.15.0.0/docs/Prelude.html#v:abs) function, so you won't need to check condition 🙂